site/docs/components/file-drop-zone/accessibility.mdx

```diff
@@ -0,0 +1,38 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Best practices
```

Comment:
These best practices don't seem to be relevant to the component's accessibility. Are these better placed in usage?